packages/webapp/src/_app/api/interfaces.ts

```diff
@@ -9,3 +11,30 @@ export enum MemberStatus {
     PendingMembership = 0,
     ActiveMember,
 }
+
+interface ElectionCompletedRound {
+    participants: EdenMember[]; // .length will be number of participants and empty if no round happened
+    participantsMemberData: MemberData[];
+    didReachConsensus?: boolean;
+    delegate?: string;
+}
+
+export interface Election {
+    member?: {
+        memberRank?: number; // undefined if still participating
+        isMemberRegisteredForElection?: boolean;
+        isMemberStillParticipating?: boolean;
+    };
+    isElectionInProgress?: boolean;
+    inProgressRoundIndex?: number; // undefined if no round (or election) in progress
+    highestRoundIndexInWhichMemberWasRepresented?: number; // no ?
+    areRoundsWithNoParticipation: boolean;
+    // .length === number of rounds that have completed (regardless of current member's participation)
+    completedRounds: ElectionCompletedRound[];
+    ongoingRound: {
+        participants: EdenMember[];
+        participantsMemberData: MemberData[];
+        projectedDelegate?: string;
+        isSortitionRound?: boolean;
+    };
+}
```

Comment:
We should consider moving these to `src/elections/interfaces`. I think the only reason `ElectionParticipationStatus` ended up here is that it's applied to members and not exclusively elections. Maybe there's a case to be made for moving it too, though.